CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## 0.2.0
+* Fixed Discovery mode to add metadata and other missing fields in catalog.json
+* Fixed Sync mode to extract data
+* Changed consumer profile endpoint to extract data in bulk mode
```

Comment:
This should be done as part of Bump version. Please remove this change for right now